neqo-transport/src/cc/classic_cc.rs

```diff
@@ -123,6 +128,20 @@ impl<T: WindowAdjustment> CongestionControl for ClassicCongestionControl<T> {
 
     // Multi-packet version of OnPacketAckedCC
     fn on_packets_acked(&mut self, acked_pkts: &[SentPacket]) {
+        // Check whether we are app limited before acked packets are removed
+        // from bytes_in_flight.
+        let is_app_limited = self.app_limited();
+        qtrace!(
+            [self],
+            "app limited={}, bytes_in_flight:{}, cwnd: {}, slow_start: {} pacing/-burst_size: {}",
+            is_app_limited,
+            self.bytes_in_flight,
+            self.congestion_window,
+            self.state.is_slow_start(),
```

Comment:
Why not just print the state?

```suggestion
            "app limited={}, bytes_in_flight:{}, cwnd: {}, state: {:?} pacing/-burst_size: {}",
            is_app_limited,
            self.bytes_in_flight,
            self.congestion_window,
            self.state,
```